packages/instant/src/components/zero_ex_instant_container.tsx

```diff
@@ -25,6 +26,7 @@ export const ZeroExInstantContainer: React.StatelessComponent<ZeroExInstantConta
         >
             <Flex direction="column" justify="flex-start">
                 <SelectedAssetInstantHeading />
+                <SelectedAssetSimulatedProgressBar />
```

Comment:
this should only show if we're in a processing, success, or failure state right?

Reply:
Yes, see https://github.com/0xProject/0x-monorepo/blob/229f11f164ce5109b771295d9aee8ebb74314181/packages/instant/src/containers/selected_asset_simulated_progress_bar.tsx#L18